### PR TITLE
Fix secrole column and time to ogc logs

### PR DESCRIPTION
**Pull request by Vampouille**

### CODE REVIEW

**Comment by fvanderbiest** [comment]
This is for https://github.com/georchestra/georchestra/issues/1279

File: ogc-server-statistics/src/main/java/org/georchestra/ogcservstatistics/dataservices/InsertCommand.java [hunk 1]
```diff
@@ -82,11 +81,6 @@ public void execute() throws DataCommandException {
             pStmt = prepareStatement();
             int updatedRows = pStmt.executeUpdate();
```
Comment (fvanderbiest) [inline ogc-server-statistics/src/main/java/org/georchestra/ogcservstatistics/dataservices/InsertCommand.java]: no need to have `updatedRows` if unused, then ?

**Comment by fvanderbiest** [comment]
LGTM

File: ogc-server-statistics/src/main/java/org/georchestra/ogcservstatistics/dataservices/InsertCommand.java (hunk 1 above)
Comment (Vampouille) [inline ogc-server-statistics/src/main/java/org/georchestra/ogcservstatistics/dataservices/InsertCommand.java]: yes, you're right

**Pull request merged**